be/src/storage/lake/lake_local_persistent_index.h

```diff
@@ -30,13 +30,21 @@ class LakeLocalPersistentIndex : public PersistentIndex {
 public:
     explicit LakeLocalPersistentIndex(std::string path) : PersistentIndex(path) { _path = path; }
 
-    ~LakeLocalPersistentIndex() override {}
+    ~LakeLocalPersistentIndex() override;
 
     Status load_from_lake_tablet(starrocks::lake::Tablet* tablet, const TabletMetadata& metadata, int64_t base_version,
                                  const MetaFileBuilder* builder);
 
+    void set_enable_persistent_index(bool enable_persistent_index) {
+        _enable_persistent_index = enable_persistent_index;
+    }
+
 private:
     std::string _path;
+
+    bool _enable_persistent_index = true;
```

Comment:
enabled by default but _tablet_id is not initialized, so if this Index instance is created and then destroyed without `load_from_lake_tablet` call, what will happen?

Reply:
`_enable_persistent_index` is true, in destructor of LakeLocalPersistentIndex, nothing will happen.